snippets/snippets.service.js

```diff
@@ -4,6 +4,20 @@ const snippetsService = {
   getAll: () => {
     const snippets = snippetsRepository.findAll();
     return snippets;
+  },
+  create: (snippetData) => {
+
+    const title = snippetData.title;
```

Comment:
Can you test sending an empty string? Reading the code seems like a bug 🐞 

You can also take advantage of arrays to do this kind of validations